### PR TITLE
AddNewBtn LA

### PR DESCRIPTION
**Pull request by hmcts-shajahan**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/services/managedocuments/ManageDocumentsService.java [hunk 1]
```diff
@@ -151,7 +151,6 @@ public Map<String, Object> copyDocument(CallbackRequest callbackRequest, String
             } else {
                 caseDataUpdated.remove(MANAGE_DOCUMENTS_RESTRICTED_FLAG);
             }
-
```
Comment (yogendraU) [inline src/main/java/uk/gov/hmcts/reform/prl/services/managedocuments/ManageDocumentsService.java:154]: can you please revert this change?

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/prl/services/managedocuments/ManageDocumentsService.java (hunk 1 above)
Comment (hmcts-shajahan) [reply src/main/java/uk/gov/hmcts/reform/prl/services/managedocuments/ManageDocumentsService.java]: Done
